src/test/java/org/jabref/logic/integrity/BooktitleCheckerTest.java

```diff
@@ -0,0 +1,30 @@
+package org.jabref.logic.integrity;
+
+import java.util.Optional;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotEquals;
+
+public class BooktitleCheckerTest {
+
+    private BooktitleChecker checker;
+
+    @BeforeEach
+    public void setUp() {
+        checker = new BooktitleChecker();
```

Comment:
No need for the `@BeforeEach` here. Just initialize the `checker` in line 13. --> One SLOC insted of 6 for the same functionality. -- JUnit initializes the class on each run.

Please check the other (modified) classes, too.